### PR TITLE
Update prow to v20190523 832050b39

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

**Comment by rmohr** [comment]
/approve
/lgtm

@cynepco3hahue changes are rolled out and job configs are valid.

File: github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ presubmits:
     optional: true
     decorate: true
     decoration_config:
-      timeout: 14400000000000 # 4h
-      grace_period: 500000000000 # 5m
+      timeout: 4h # 4h
```
Comment (slintes) [inline github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: nitpicking: we don't need the comments anymore...

Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: Done. :+1:

**Pull request merged**